Feature/2927 edit online offline calibration

cw-cvp opened the pull request.

cw-cvp committed:
add update online offline calibration endpoint

cw-cvp committed:
add updateOnlineOfflineCalibration save data switch case

cw-cvp committed:
remove aps indicators from dropdown array in props

cw-cvp committed:
add else block for defaultValue

cw-cvp committed:
update user input extraction to use getListOfRadioControls

addisgmichael commented on src/additional-functions/create-modal-input-controls.js:
```diff
@@ -209,24 +209,22 @@ export const modalViewData = (
           case "radio":
             const curInputOptions = inputs[y][4]
-            if (curInputOptions) {
+            if (selected) {
```
Can you pls confirm these updates do not break already implemented radio controls?

cw-cvp replied:
I can confirm these changes do not break already implemented radio controls. I added a 5th index to the input controls for qa-datatable-props for online offline calibration.
The Online Zero/Online Upscale/Offline Zero/Offline Upscale APS Indicators for Online Offline Calibration use this 5th index for a default value of yes for the radio button selection.

addisgmichael replied:
Ok, thanks,

cw-cvp committed:
fix issues

-table displays yes/no instead of 1/0 for radio buttons (aps indicator controls)
-fixed typo causing table not to display upscale gas level code
